### PR TITLE
feat: add positive and negative tests for MPT inclusion

### PR DESCRIPTION
**Pull request by MonkeyKing-1**

### CODE REVIEW

File: scripts/input_gen/INFURA_ID [hunk 1]
```diff
@@ -0,0 +1 @@
+0c602f5267ba478eb5b101f1f1b335d4
```
Comment (yi-sun) [inline scripts/input_gen/INFURA_ID]: Remove this private file.

File: src/mpt/tests/mpt_tests.rs [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    cell::RefCell,
+    env::{set_var, var},
+    fs::File,
+    io::{BufReader, Write},
+    path::Path,
+};
+use test_log::test;
+use test_case::test_case;
+
+// fn get_test_circuit(network: Network, num_slots: usize) -> EthBlockStorageCircuit {
```
Comment (yi-sun) [inline src/mpt/tests/mpt_tests.rs]: Is this intentionally commented out?  If so, remove

File: src/mpt/tests/mpt_tests.rs [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    let params = EthConfigParams::from_path("configs/tests/mpt.json");
+    let k = params.degree;
+    let input = mpt_input("scripts/input_gen/noninclusion_extension_pf.json", true, 6); // require depth < max_depth
+    let circuit = test_mpt_circuit(k, RlcThreadBuilder::<Fr>::mock(), input);
+    MockProver::run(k, &circuit, vec![]).unwrap().assert_satisfied();
+}
+
+
+
+//"scripts/input_gen/noninclusion_extension_pf.json"
```
Comment (yi-sun) [inline src/mpt/tests/mpt_tests.rs]: Clean up all comments below too

**Review by yi-sun — CHANGES_REQUESTED** [review]

File: scripts/input_gen/INFURA_ID (hunk 1 above)
Comment (MonkeyKing-1) [reply scripts/input_gen/INFURA_ID]: Done

**Comment by MonkeyKing-1** [comment]
Comments and key are both removed now

File: src/mpt/tests/mpt_tests.rs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+#[test_case("scripts/input_gen/pos_data/inclusion1_pf.json".to_string(); "correct inclusion 1")]
+#[test_case("scripts/input_gen/pos_data/inclusion2_pf.json".to_string(); "correct inclusion 2")]
+#[test_case("scripts/input_gen/neg_data/wrong_path_default_storage_pf.json".to_string(); "wrong path inclusion")]
+#[test_case("scripts/input_gen/neg_data/wrong_val_default_storage_pf.json".to_string(); "wrong val inclusion")]
+#[test_case("scripts/input_gen/neg_data/wrong_proof_default_storage_pf.json".to_string(); "wrong proof inclusion")]
+
+pub fn test_mpt_inclusion_fixed(path : String) {
+    let params = EthConfigParams::from_path("configs/tests/mpt.json");
+    // std::env::set_var("ETH_CONFIG_PARAMS", serde_json::to_string(&params).unwrap());
```
Comment (yi-sun) [inline src/mpt/tests/mpt_tests.rs:46]: Remove extraneous comments

**Review by yi-sun — APPROVED** [review]
Fine to merge after removing the last extraneous comment.

**Pull request closed without merging**